backend/app/usecase/risk/blacklist_fake.go

```diff
@@ -0,0 +1,26 @@
+package risk
+
+var _ BlackList = (*BlackListFake)(nil)
+
+var blacklist = map[string]struct{}{
+	"http://malware.wicar.org/data/ms14_064_ole_not_xp.html": struct {
+	}{},
```

Comment:
Nit: Format looks wired.

Reply:
Agreed I changed it to `map[string]bool` so that it doesn't require an empty struct.